docs/markdown/Reference-manual.md

```diff
@@ -261,6 +261,9 @@ following.
 - `build_by_default` *(added 0.38)* causes, when set to true, to
   have this target be built by default, that is, when invoking plain
   `ninja`; the default value is false
+  *(added 0.49)* if `build_by_default` is explicitly set to false, `install`
```

Comment:
I think this should be `changed in 0.49` instead of `added`

Reply:
sure, done